classes/ShoppingfeedApi.php

```diff
@@ -502,11 +502,10 @@ public function acknowledgeOrder($id_order_marketplace, $name_marketplace, $id_o
                 throw new Exception('Invalid store ID');
             }
 
-            $operation = new ShoppingFeed\Sdk\Api\Order\OrderOperation();
+            $operation = new ShoppingFeed\Sdk\Api\Order\Operation();
             $operation
                 ->acknowledge(
-                    (string) $id_order_marketplace,
-                    (string) $name_marketplace,
+                    new Id($id_order_marketplace),
```

Comment:
$id_order_marketplace is not unique
For me the parameter should be $id_internal_shoppingfeed

Reply:
Actually, the variable $id_order_marketplace contains a value of internal shoppingfeed order id. Or you want me to rename the variable?

Reply:
OK, in this case the name $id_internal_shoppingfeed is improper.
Could you please change the name by the right name of the object model ?
https://github.com/shoppingflux/module-prestashop/blob/develop/classes/ShoppingfeedOrder.php#L30C13-L30C37
In the class ShoppingfeedApi, ligne 297, the parameter is named id_internal_shoppingfeed

Reply:
variable is renamed